source/includes/cxi/_segments.md

```diff
@@ -0,0 +1,45 @@
+# Segments/Properties
+
+### HTTP Request
+
+`POST https://cxi-api.wootric.com/v1/segments`
+
+## Response segments bulk update
+
+Parameter | Type | Description
+--------- | ---- | -------
+resposes[id] | integer | Response ID
+response[custom_properties] | Array | list of properties(Hash { key: property_name, value: new_value})
+response[custom_properties][][key] | string | name of the segment/property
+response[custom_properties][][value] | string | new value for the segment/property
+
+#### Update bulk
```

Comment:
This is here and in the section below. It seems to be the same text `This endpoint updates a list of segments from a response or from a user`. Is that right?

Reply:
Updated, thanks!